src/mmrelay/meshtastic/packet_routing.py

```
@@ -0,0 +1,179 @@
+
+class PacketAction:
+    RELAY = "relay"
+    PLUGIN_ONLY = "plugin_only"
+    DROP = "drop"
+
+
+CHAT_ELIGIBLE_PORTNUMS = frozenset({TEXT_MESSAGE_APP})
+logger = get_logger("Meshtastic")
+
+
+def _get_portnum_name(portnum: Any, packet: dict[str, Any] | None = None) -> str:
```

Ruff (0.15.9)

[warning] Too many return statements (8 > 6)

(PLR0911)